modules/claims_api/spec/requests/v1/disability_compensation_request_spec.rb

```diff
@@ -1327,6 +1327,20 @@ def obj.class
       end
     end
 
+    context 'disability name starts/ends with a speace' do
+      let(:data) do
+        temp = File.read(Rails.root.join('modules', 'claims_api', 'spec', 'fixtures', 'form_526_json_api.json'))
+        temp = JSON.parse(temp)
+        temp['data']['attributes']['autoCestPDFGenerationDisabled'] = auto_cest_pdf_generation_disabled
+        temp['data']['attributes']['claimDate'] = claim_date
+        temp['data']['attributes']['applicationExpirationDate'] = (Time.zone.today + 1.day).to_s
+        temp['data']['attributes']['disabilities'][0]['name'] =
+          " #{temp['data']['attributes']['disabilities'][0]['name']} "
+
+        temp.to_json
+      end
+    end
```

Comment:
I don't think ya need this.
It doesn't look like there's any spec logic that is making any expectations on that stripping behavior.